editions/tw5.com/tiddlers/howtos/How to embedd PDF and other documents.tid

```diff
@@ -0,0 +1,23 @@
+created: 20141117000000000
+modified: 20161228201311874
+tags: Learning
+title: How to embedd PDF and other documents
+type: text/vnd.tiddlywiki
+
+There are two ways to embed a PDF in your TW5 file.
```

Comment:
I think this article should perhaps start öile ...

''Note:'' Before we dive into how you can embed files of a certain type, you may want to consider not to and simply link external resources using `[ext[title|path/to/that.file]]` and let your browser decide how to handle the resource.

There are two ways to embed a PDF in your TW5 file,